client-templates/github-com/accept.json.sample

```diff
@@ -90,6 +90,14 @@
           {
             "path": "commits.*.modified.*",
             "value": "packages.config"
+          },
+          {
+            "path": "commits.*.added.*",
+            "value": "*.csproj"
+          },
+          {
+            "path": "commits.*.modified.*",
+            "value": "*.csproj"
```

Comment:
I wonder if the filter validation treats `*` as wildcards in the `value` field as well.

Reply:
testing locally, will confirm shortly

Reply:
Verified locally, all events looked good nothing blocked when testing with local webhooks setup.